Formula/qpid-proton.rb

```diff
@@ -0,0 +1,64 @@
+class QpidProton < Formula
+  desc "is a high-performance, lightweight AMQP 1.0 messaging library."
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www-eu.apache.org/dist/qpid/proton/0.17.0/qpid-proton-0.17.0.tar.gz"
+  sha256 "6ffd26d3d0e495bfdb5d9fefc5349954e6105ea18cc4bb191161d27742c5a01a"
+
+  depends_on "cmake" => :build
+  depends_on "libuv"
+  depends_on "openssl"
+  depends_on "perl" if Formula["perl"].installed?
+  depends_on "python" if Formula["python"].installed?
+
+  def install
+    # Javascript bindings switched off - leads to build errors
+    args = %w[
+      -DBUILD_JAVASCRIPT=OFF
+    ]
+
+    if MacOS.version == "10.11" && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
+      inreplace "proton-c/CMakeLists.txt",
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!